src/ProductConstructionService/ProductConstructionService.Api/Queue/QueueConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace ProductConstructionService.Api.Queue;
+
+public static class QueueConfiguration
+{
+    public const string PcsJobQueueName = "PcsJobQueueName";
```

Comment:
This should really be `JobQueueConfigurationKey` or something so it's clear this is not what the queue is named

Reply:
Makes sense